dss/events/handlers/notify_v2.py

```diff
@@ -0,0 +1,178 @@
+import os
+import json
+import requests
+import urllib3
+from requests_http_signature import HTTPSignatureAuth
+import logging
+from uuid import uuid4
+from collections import defaultdict
+
+from functools import lru_cache
+import jmespath
+from jmespath.exceptions import JMESPathError
+import boto3
+
+import dss
+from dss import Config, Replica
+from dss.util.aws.clients import dynamodb, sqs  # type: ignore
+from dss.subscriptions_v2 import SubscriptionData, get_subscriptions_for_replica
+
+logger = logging.getLogger(__name__)
+
+notification_queue_name = "dss-notify-v2-" + os.environ['DSS_DEPLOYMENT_STAGE']
+subscription_ddb_table = "bhannafi-test-subscriptions-" + os.environ['DSS_DEPLOYMENT_STAGE']
```

Comment:
I'm guessing `"bhannafi-test-subscriptions-"` is temporary while this is a WIP.

Reply:
thanks, that was cruft. removed